docs/source/examples/dispatch_tutorial.ipynb

```diff
@@ -0,0 +1,513 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Dispatch Tutorial\n",
+    "\n",
+    "In this tutorial you will learn how to run a dispatch model in `osier` using\n",
+    "1. `osier` technology objects\n",
+    "2. The `osier` dispatch algorithm\n",
+    "\n",
+    "\n",
+    "First, we must import some key ingredients."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# basic imports\n",
+    "import pandas as pd \n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from unyt import kW, minute, hour, day, MW\n",
+    "\n",
+    "# osier imports\n",
+    "from osier import DispatchModel\n",
+    "import osier.tech_library as lib"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating the technology portfolio\n",
+    "\n",
+    "In order to run a dispatch model you must provide a mix of energy generating technologies. The technologies used in the dispatch model should be added to a python *list*. \n",
+    "\n",
+    "In general, **dispatch models do not optimize the capacity of each generator.** They only optimize the amount of energy each generator produces (i.e., dispatches). \n",
+    "\n",
+    "If you forget what technologies are available in `osier` (or what they're called), simply look at the catalog!"
+   ]
```

Comment:
You may be able to leave out explicitly showing the catalog in this tutorial, since you show it on the technology tutorial page.

Reply:
I wanted to remind readers that this is a thing they can do! Instead of memorizing technology names...